csi/util.go

```diff
@@ -25,16 +27,16 @@ const (
 	defaultStaleReplicaTimeout = 2880
 )
 
-// NewForcedParamsOsExec creates a osExecutor that allows for adding additional params to later occurring Run calls
-func NewForcedParamsOsExec(cmdParamMapping map[string]string) mount.Exec {
+// NewForcedParamsExec creates a osExecutor that allows for adding additional params to later occurring Run calls
+func NewForcedParamsExec(cmdParamMapping map[string]string) utilexec.Interface {
 	return &forcedParamsOsExec{
-		osExec:          mount.NewOSExec(),
+		exec:            utilexec.New(),
 		cmdParamMapping: cmdParamMapping,
 	}
 }
```

Comment:
Can you double-check this change @joshimoo ? Kubenetes v1.18 moves the mount package around and I have to make this change

Reply:
Looks good, god job on the change. With them moving this stuff into a dedicated util package we can potentially replace the nsenter exec after 1.2 and push some changes to the util package.